src/User.php

```diff
@@ -4438,17 +4438,15 @@ public static function dropdown($options = [])
         }
 
         // Make a select box with all glpi users
+        $view_users = self::canView();
+
         if (!$p['multiple']) {
             $user = getUserName($p['value'], 2, true);
-        }
```

Comment:
i can't understand why the variable was undefined, and also, your change removed a closing brace, no ?

Reply:
`$user` is only defined in `if (!$p['multiple']) {` block, so I move its usage in this block. Then, as there was 2 `if (!$p['multiple']) {` blocks following each other, I just merged them.